media/js/base/protocol/m24-navigation.es6.js

```diff
@@ -122,8 +122,8 @@ MzpNavigation.initSticky = () => {
  * user scrolls up and down the viewport.
  */
 MzpNavigation.checkScrollPosition = () => {
-    // add styling for when scrolling the viewport
-    if (window.scrollY > 0) {
+    // add styling for when scrolling the viewport and nav is already sticking
+    if (window.scrollY > 0 && _navElem.getBoundingClientRect().top == 0) {
```

Comment:
E.g. I'd add some position tolerance instead of hard y===0, given the potential subpixel precision of this or some "rubberband" overscroll etc:

```suggestion
    if (window.scrollY > 0 && _navElem.getBoundingClientRect().top < 1) {
```

Reply:
I have been considering Intersection Observers but that felt like needing a little different approach for most of the `MzpNavigation.*Sticky` logic, so this was to minimize the change surface. Observer API would be a better approach generally, so if this feels wonky at times that might be the next iteration.